doc/book/admin/start_stop_instance.rst

```diff
@@ -3,335 +3,387 @@
 Starting and stopping instances
 ===============================
 
-To start a Tarantool instance from an :ref:`instance file <admin-instance_file>`
-using the :ref:`tt <tt-cli>` utility:
+This section describes how to manage instances in a Tarantool cluster using the :ref:`tt <tt-cli>` utility.
+A cluster can include multiple instances that run different code.
+A typical example is a cluster application that includes router and storage instances.
+For example, you can manage instances in the following ways:
```

Comment:
"manage instances in the following ways" sounds a bit awkward. Maybe something more goal-oriented? Or just: "Particularly, you can perform the following actions:"